master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Securing access to Calico's metrics endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with prometheus metrics enabled, it may be desirable to use network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
```

Comment:
prometheus as a proper noun should be capitalized, throughout.

This introductory sentence could be a little more interesting: it could mention something about why. It might be nice to  help them a little bit with their decision. Undoubtedly, this task will involve some effort, so it's good to give them a little motivation for their labors. Is it because some sensitive data gets transmitted? I actually do not know either why they might want to do it. Also, I do not know if we might want to go so far as to recommend it? Up to you....